visualization/app/codeCharta/util/nodeDecorator.spec.ts

```diff
@@ -23,7 +15,7 @@ describe("nodeDecorator", () => {
 
 	beforeEach(() => {
 		file = clone(TEST_DELTA_MAP_A)
-		map = file.map
+		;({ map } = file)
```

Comment:
ew, that's ugly. I'd rather read the old style. :b

Reply:
Can't we do it like this?
```ts
		file = clone(TEST_DELTA_MAP_A);
	        ({ map } = file)
```

Reply:
I agree and I prefer the old style as well. I'll fix that in the config.